sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/encrypt_parameters.hpp

```diff
@@ -24,8 +24,17 @@ namespace Azure {
    * @brief Parameters for encrypting plaintext.
    *
    */
-  struct EncryptParameters final
-  {
+  class EncryptParameters final {
+  private:
+    /**
+     * @brief Gets the initialization vector for encryption.
+     *
+     * @note Initialization vector should not be set for some encryption algorithms. That's why it
+     * is private so it is only set by the factory methods.
+     *
+     */
+    std::vector<uint8_t> Iv;
```

Comment:
private fields should be `m_`.

```suggestion
    std::vector<uint8_t> m_iv;
```